tardis/io/atom_data/util.py

```diff
@@ -35,9 +35,9 @@ def resolve_atom_data_fname(fname):
         )
         return fpath
 
-    atom_data_name = fname.replace(".h5", "")
+    atom_data_name = fname.with_suffix("")
```

Comment:
Since fname is an absolute path coming from "atom_data_fname" in simulation/base.py in stead of the file name only, should we use `fname.stem` here as suggested by https://github.com/tardis-sn/tardis/pull/2535#issuecomment-1987317427?